nbqa/config.py

```diff
@@ -18,6 +23,31 @@ class _ConfigSections(NamedTuple):  # pylint: disable=R0903
 CONFIG_SECTIONS = _ConfigSections()
 
 
+try:
+    ISORT_MODULE: Optional[ModuleType] = import_module("isort")
+except ImportError:  # pragma: nocover (this case is tested in test_isort_works.py)
+    ISORT_MODULE = None
+
+
+def get_default_configs() -> Mapping[str, List[str]]:
```

Comment:
The option `--treat-comment-as-code` is only available for isort 5.3.0 onwards, so I think we would have to try importing `isort` to tell whether it's of a modern enough version or not

Reply:
If you want to include a data file, make sure to properly include it with `package_data` (been there done that, shipping a broken package because of a missing data file 😜 )
Here is a nice video explaining the datafile packaging differences
[![](http://img.youtube.com/vi/OXmYKh0eTQ8/0.jpg)](http://www.youtube.com/watch?v=OXmYKh0eTQ8 "")

Reply:
all praise our saviour Anthony Sottile

Reply:
@s-weigand is that the video you meant to link? I couldn't figure out how to package a toml file from it, sorry :rofl: